doc/api/deprecations.md

```diff
@@ -998,7 +998,7 @@ deprecated and support will be removed in the future.
 
 Type: Documentation-only
 
-The option `produceCachedData` has been deprecated. Use
+The option `produceCachedData` is deprecated. Use
```

Comment:
Micro-nit, can be ignored: There are five instances of _`suchAndSuch` option_ and this is the only instance of _option `suchAndSuch`_. Change this to _`produceCacheData` option_ rather than _option `produceCacheData`_.